test/multiverse/suites/memcache/Envfile

```diff
@@ -1,15 +1,22 @@
 require File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "..", "helpers", "docker"))
 
-# TODO: Adjust instrumentation/tests to support latest version of dalli
-# if RUBY_VERSION >= '2.6.0'
-#   gemfile <<-RB
-#     gem 'dalli'
-#   RB
-
-#   gemfile <<-RB
-#     gem 'dalli', '3.2.1'
-#   RB
-# end
+if RUBY_VERSION >= '2.6.0'
+  gemfile <<-RB
+    gem 'dalli'
+  RB
+
+  gemfile <<-RB
+    gem 'dalli', '3.2.0'
+  RB
+
+  gemfile <<-RB
+    gem 'dalli', '3.1.6'
+  RB
+
+  gemfile <<-RB
+    gem 'dalli', '3.1.0'
+  RB
```

Comment:
out of curiosity, why did you choose these specific versions?

Reply:
The jump from 3.0.6 -> 3.1.0 and the jump for 3.1.0 -> 3.1.1 were the version bumps that had the breaking changes for the agent. I wanted to make sure we definitely included 3.1.0 because it was different from the one immediately before and immediately after. 
3.1.1 up to 3.1.6 all behaved the same way with the agent, so i just grabbed the latest in that group.
Then I also grabbed 3.2.0 since its the next minor bump to make sure we're testing that. I didn't want to add every version, since there would be 10 new runs and that would add so much time. 
Then I left the last one unpinned to always grab latest.
Hopefully that makes sense! Also I'd be down to change this up some if you have any suggestions!

Reply:
Tanna would the extra context you provided here typically be a good use case for a comment? Or can it usually be assumed if I saw something similar like this in the future that there are likely key differences between these versions.

Reply:
thats a good idea, i added some comments in there to do that. thank you!